seacatauth/events.py

```diff
@@ -0,0 +1,28 @@
+class EventTypes:
+	EVENT_TYPE = "event_type"
+	LOGIN_SESSION_CREATED = "login_session_created"
+	LOGIN_SESSION_UPDATED = "login_session_updated"
+
+	WEBAUTHN_CREDENTIALS_CREATED = "webauthn_credentials_created"
+	WEBAUTHN_CREDENTIALS_UPDATED = "webauthn_credentials_updated"
+	REGISTRATION_CHALLENGE_CREATED = "registration_challenge_created"
```

Comment:
`WEBAUTHN_REG_CHALLENGE_CREATED = "webauthn_registration_challenge_created"`
i know it's a bit long but there will probably be more types of registration challenges